administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-06-14.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE if NOT EXISTS  `#__share_draft`(
+  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key to #__content',
+  `title` varchar(255) NOT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL, 
+  `sharelink` varchar(255) NOT NULL, 
+  PRIMARY KEY  (`id`)
+) DEFAULT CHARSET=utf8;
```

Comment:
@nikitadhiman This looks good :+1: The other files have two many empty lines at the end. Once that is corrected, we are good to go.